install_ffmpeg.sh

```diff
@@ -84,9 +81,7 @@ if [[ $(uname) != *"MSYS"* ]]; then
   if [ ! -e "$HOME/gnutls-3.5.18" ]; then
     cd $HOME
     curl -LO https://www.gnupg.org/ftp/gcrypt/gnutls/v3.5/gnutls-3.5.18.tar.xz
-    xz -d gnutls-3.5.18.tar.xz
-    tar xf gnutls-3.5.18.tar
-    rm gnutls-3.5.18.tar
```

Comment:
I don't have a strong preference, but should we remove the archive after decompressing?

Reply:
So far, we haven't been removing the downloaded files - only the intermediate tar files. Right now, we don't produce those intermediate files anymore, so that's why those lines were removed.

Only a slight preference for keeping the downloaded files - the pack rat in me likes to have the downloaded files, just because you never know if they might be handy some day (suppose the archive disappears from the Internet!) but I don't really have a better argument in favor of it. Happy to remove if others are more inclined that way.

I've been debating whether to move the working / install directory for the `install_ffmpeg.sh` script into `go-livepeer/deps` just so the downloaded / extracted stuff would be out of the way and self-contained within `deps`. Eg, `HOME=$PWD`. That might break CI caching without a bit more work though - to the extent that "CI caching" actually happens anymore (we used to attempt caching of ffmpeg and friends to improve CI run times).